src/lib/EventSubscriber/AppRendererSubscriber.php

```diff
@@ -51,7 +51,7 @@ public function renderApp(FilterResponseEvent $event)
 
         $response = $event->getResponse();
 
-        if ($response->isRedirect()) {
+        if ($response->isRedirect() || $response instanceof NoRenderResponse) {
```

Comment:
Let's keep it as is for the time being. Changes are necessary in that area anyway.